CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ### Bug fixes
 
+### Deprecated functionality
+* Deprecated `skbio.util.flatten`. Please use standard python library functionality
+described here (http://stackoverflow.com/questions/952914/making-a-flat-list-out-of-list-of-lists-in-python)
```

Comment:
Can you please format these stackoverflow links using markdown syntax? Also add the version of scikit-bio that it will be removed in.